test/Microsoft.AspNet.Mvc.IntegrationTests/ValidationIntegrationTests.cs

```diff
@@ -997,14 +997,14 @@ private class Order10
 
             // Assert
             Assert.NotNull(modelBindingResult);
-            Assert.False(modelBindingResult.IsModelSet);
+            Assert.True(modelBindingResult.IsModelSet);
 
             var model = Assert.IsType<List<Order10>>(modelBindingResult.Model);
             Assert.Empty(model);
 
             Assert.Equal(0, modelState.Count);
             Assert.Equal(0, modelState.ErrorCount);
-            Assert.False(modelState.IsValid);
+            Assert.True(modelState.IsValid);
```

Comment:
This was just a bad copy-paste I think. Because there's no data, there's nothing to be invalid.